src/commands/misc/infos.js

```diff
@@ -1,7 +1,10 @@
-const { version } = require('../../../package.json')
-const { diffDate } = require('../../util/util')
+import { diffDate } from '../../util/util.js'
 
-module.exports = {
+// import nodePackage from '../../../package.json'
+import { readFileSync } from 'fs'
+const { version } = JSON.parse(readFileSync('./package.json'))
```

Comment:
```suggestion
import nodePackage from '../../../package.json'
```
Utilisation d'import possible uniquement lors de l'utilisation du flag `--experimental-json-modules`. Pour l'instant, cette feature est à l'état expérimental (stability: 1) : https://nodejs.org/api/esm.html#esm_json_modules